python-sdk/example_dags/example_transform_file.py

```diff
@@ -1,4 +1,3 @@
-import pathlib
 from datetime import datetime
```

Comment:
Should we remove this file since the function is deprecated and we ask users to use `transform_sql` instead?

Reply:
Also, why a new name `transform_sql`, why not just change the code of `transform_file`?

Reply:
I am not sure. We also keep the docs and just add a note about that it is deprecated. Also we use the example dag to test that the deprecated function still works. Deprecating a function does not mean we should not test it in my opinion.

Reply:
I agree with @feluelle that we should continue testing the function even if it is deprecated - since it is still part of our interface, and we don't want for it to break until we remove it from the source-code.

My understanding was that this was a breaking change. If it is not a breaking change, I'm in favour of keeping `transform_file`. Did I get it wrong, @feluelle ? :see_no_evil:

Reply:
🙈 all the work for nothing 🤦  I'll revert the `transform_sql` as we can also use `transform_file` in the sql-cli now. I added this because I thought we need a `transform_sql`.

Reply:
Fixed. PTAL @kaxil @tatiana

Reply:
Yeah +1 on keeping the name `transform_file`, to me that's descriptive of what it's doing.